pkg/dogstatsd/server.go

```diff
@@ -446,8 +449,11 @@ func (s *Server) ServerlessFlush() {
 	for _, w := range s.workers {
 		w.flush()
 	}
-	s.aggregator.ServerlessFlush <- true
-	<-s.aggregator.ServerlessFlushDone
+	// flush the aggregator to have the serializer/forwarder send data to the backend.
+	// We add 10 seconds to the interval to ensure that we're getting the whole sketches bucket
```

Comment:
Where is this addition of 10s?

Reply:
Fair, let me remove this part of the comment.

(For history, the part adding the 10seconds was initially there when I've implemented this for the serverless agent, the serverless team has moved the code into the Aggregator [here](https://github.com/DataDog/datadog-agent/blob/7.33.x/pkg/aggregator/aggregator.go#L762))